.travis.yml

```diff
@@ -13,10 +13,3 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-4.8
-
-services:
```

Comment:
why do you remove these lines?

Reply:
I was following the example of https://github.com/Wolox/express-js-bootstrap/blob/master/.travis.yml given in the trello card, but then i double checked with a trainer and added everything that was missing for travis